docs/source/configuration/settings-reference.md

```diff
@@ -374,8 +374,17 @@ additionalToolbarComponents
       }}
     </Plug>
     ```
-```
+````
```

Comment:
This is the closing fence for the opening glossary fence at the start of the page. It should be moved after your new glossary entry.

```suggestion
```